### PR TITLE
E 1078 clean up sumsub sdk integration

### PR DESCRIPTION
**Pull request by haydencleary**

### CODE REVIEW

File: app/api/sumsub-token/route.ts [hunk 1]
```diff
@@ -1,212 +1,87 @@
...
-  const method = "post";
-  const url =
+  config.url =
     "/resources/accessTokens?userId=" +
-    encodeURIComponent(externalUserId) +
+    encodeURIComponent(externalId) +
     "&ttlInSecs=" +
     ttlInSecs +
     "&levelName=" +
     encodeURIComponent(levelName);
```
Comment (alexbeno) [inline app/api/sumsub-token/route.ts:41]: can we use template string ?

File: app/api/sumsub-token/route.ts [hunk 2]
```diff
@@ -1,212 +1,87 @@
...
-// Such actions are presented below:
-// 1) Creating an applicant
-// 2) Adding a document to the applicant
-// 3) Getting applicant status
-// 4) Getting access tokens for SDKs
-
 export async function POST(request: Request) {
-  const { userId: externalUserId, levelName = "basic-kyc-level" } = await request.json();
-  console.log("External UserID: ", externalUserId);
+  const { externalId, levelName = SUMSUB_CONST.DEFAULT_LEVEL } = await request.json();
```
Comment (alexbeno) [inline app/api/sumsub-token/route.ts:52]: you don't need to catch the request.json() ?

File: src/App/Stacks/VerifySidePanel/VerifySidePanel.tsx [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
 import { useEffect, useState } from "react";
 
+import { createSumsubToken } from "app/api/sumsub-token/handlers";
+
+import { TVerifySidePanel } from "src/App/Stacks/VerifySidePanel/VerifySidePanel.types";
+import { IMAGES } from "src/assets/img";
+
+import { EmptyState } from "components/layout/placeholders/empty-state";
+
 const config = {
```
Comment (alexbeno) [inline src/App/Stacks/VerifySidePanel/VerifySidePanel.tsx:11]: maybe we can export this config in a external file ?

Comment (haydencleary) [reply src/App/Stacks/VerifySidePanel/VerifySidePanel.tsx]: 3b12185c7e89cae30d62f79c58df255e6ad67014

File: app/api/sumsub-token/route.ts (hunk 2 above)
Comment (haydencleary) [reply app/api/sumsub-token/route.ts]: 48b702382e4297d2b38f7dfdbb451dde13d6452b

File: app/api/sumsub-token/route.ts (hunk 1 above)
Comment (haydencleary) [reply app/api/sumsub-token/route.ts]: 136bc2ec6371c281a3e5baef43afec67d5fdf1bf

File: src/App/Stacks/VerifySidePanel/VerifySidePanel.tsx [hunk 4]
```diff
@@ -156,49 +160,57 @@ const config = {
...
+  function handleError() {
+    setError(true);
+  }
+
+  if (error) {
+    return (
+      <div className="flex h-full items-center justify-center">
+        <EmptyState
+          illustrationSrc={IMAGES.icons.emptyState}
+          title={{ token: "v2.pages.settings.billing.sumsub.error.title" }}
```
[the remark this answers is not in the document]
Comment (haydencleary) [reply src/App/Stacks/VerifySidePanel/VerifySidePanel.tsx]: f2a887940a19fcce1cf73547768368d4aaedefb7

**Review by alexbeno — APPROVED** [review]

**Pull request merged**